backends/tfhe-cuda-backend/cuda/src/pbs/programmable_bootstrap_multibit.cuh

```diff
@@ -152,11 +162,23 @@ __global__ void device_multi_bit_programmable_bootstrap_accumulate_step_one(
 
   selected_memory = sharedmem;
 
+  if constexpr (SMD == FULLSM) {
+    selected_memory = sharedmem;
+  } else {
+    int block_index = blockIdx.x + blockIdx.y * gridDim.x +
+                      blockIdx.z * gridDim.x * gridDim.y;
+    selected_memory = &device_mem[block_index * device_memory_size_per_block];
+  }
+
   Torus *accumulator = (Torus *)selected_memory;
   double2 *accumulator_fft =
       (double2 *)accumulator +
       (ptrdiff_t)(sizeof(Torus) * polynomial_size / sizeof(double2));
 
+  if constexpr (SMD == PARTIALSM)
+    accumulator_fft = (double2 *)sharedmem;
+
+  //
```

Comment:
Unnecessary line :slightly_smiling_face:

Reply:
The `//`?